apps/desktop/desktop/src/main/java/bisq/desktop/main/content/authorized_role/mediator/MediatorController.java

```diff
@@ -244,7 +244,13 @@ private void update() {
 
     private void maybeSelectFirst() {
         if (!model.getListItems().getFilteredList().isEmpty()) {
```

Comment:
If we do the check in the UIThread.runOnNextRenderFrame we can remove that check.